File: server/express/src/modules/auth/auth.route.ts
import { Router } from "express";
import { authController } from "./auth.controller";
import { Request, Response, NextFunction } from "express";

const authRoute = Router();

// Middleware Endpoint Auth
// authRoute.use()

// Method
authRoute.post("/register", (req : Request, res : Response, next: NextFunction) => authController.register(req , res, next));
authRoute.post("/login", (req : Request, res : Response, next: NextFunction) => authController.login(req , res, next));
authRoute.post("/logout", (req: Request, res: Response, next: NextFunction) => authController.logout(req, res, next) );

export default authRoute;

File: server/gofiber/internal/modules/auth/auth_route.go
package auth

import (
	// "gofiber/internal/middleware"

	"github.com/gofiber/fiber/v2"
)

func AuthRoute(r fiber.Router, h *AuthHandler) {
	r.Post("/register", h.RegisterUser)
	r.Post("/login", h.LoginUser)
}
